doc/architecture.rst

```diff
@@ -369,3 +369,37 @@ is renamed or moved.
 Example:
 
   <oc:id>00000020oc5cfy6qqizm</oc:id>
+  
+Virtual Files
+----------------
+.. note::
+    * This feature is currently only available on ``Windows`` by default. ``Linux`` implementation is experimental and must be enabled by adding ``enableExperimentalOptions=true`` to the ``nextcloud.cfg`` configuration file in the ``App Data`` folder. ``macOS`` implementation is not available at the moment.
+
+Oftentimes, users are working with a huge amount of files that are big in size. Synchronizing every such file to a device that's running a Nextcloud desktop client is not always possible due to the user's device storage space limitation.
+Let's assume that your desktop client is connected to a server that has 1TB of data. Now, let's assume, you want all those files at hand, so you can quickly access any file via the file explorer. Your device has 512GB SSD storage inside.
```

Comment:
I would drop the ssd word. Probably does just confuse users and does not play a role I think:)

Reply:
@FlexW Was trying to **emphasize** the small storage space, as, usually, HDDs are much more affordable and those having the HDD may not need to use this feature as they can allow a much higher volume of data. But, OK, I got your point. Will probably drop it.